src/main/play-doc/operation/ConfigurationRef.md

```diff
@@ -780,3 +787,953 @@ cinnamon {
+          max-backoff = 30 seconds
+          random-factor = 0.2
+        }
+      }
+    }
+
+    bundle-house-keeping {
+      # Bundle house keeping will try to remove unused files from the bundles directory specified in
+      # `conductr.agent.storage-dir`. Bundle house keeping will be invoked when a new bundle starts.
+      #
+      # Bundle house keeping will attempt to delete bundle run directories except the latest directories.
+      # The number of the latest directories to be kept is controlled by `bundle-dirs-threshold`. This attempt will
+      # remove old bundle run directories and its contents (i.e. jars and logs, etc).
+      #
+      # After cleaning up the bundle run directories, and if the size threshold is breached, bundle house
+      # keeping will delete files older than specified `time-threshold`. This attempt will remove old bundle zip files.
+      #
+      # If the size threshold is still breached after this attempt, log warning message will be issued.
+
+      # Number of bundle run directory to be kept within a particular bundle.
+      # Important: never set this number below 1, otherwise the directory of currently running bundle will be deleted.
+      bundle-run-dirs-threshold = 2
+
+      size-threshold = 4GB
+      time-threshold = 30 days
+
+      # Time delay between bundle housekeeping being scheduled to when it's performed.
+      start-delay = 5 seconds
+    }
+  }
+
+  service-proxy {
+    # The name given to a service representing the proxy being used.
+    name = "conductr-haproxy"
+  }
+}
+
+# Lightbend Contrail configuration
+contrail {
+
+  syslog {
+
+    loglevel = ${akka.loglevel}
+
+    # Used as APP-NAME in syslog events.
+    app-name = "ConductR-Agent"
+
+    server {
+      # We lean on the Mesos Elasticsearch instead of one managed by ConductR
+      service-locator {
+        enabled        = on
+        selection-path = "/user/reaper/syslog-service-locator-client"
+        service-name   = "_client-port._elasticsearch-executor._tcp.elasticsearch.mesos"
+      }
+
+      elasticsearch {
+        enabled = on
+        index = "conductr"
+      }
+    }
+  }
+}
+
+# Lightbend Monitoring configuration
+cinnamon {
+
+  instrumentation = off
+
+  akka.actors {
+    "/user/*" {
+      report-by = class
+    }
+  }
+}
+
+# A connection context for running the Mesos executor driver
+mesos-executor-driver-run-executor {
+  executor = "thread-pool-executor"
+
+  # Controls the number of mesos executor drivers per node
+  # The `MesosExecutorDriver.run()` blocks the entire thread until the driver has been stopped or aborted.
+  # Therefore we decicate a separate dispatcher to the run() method.
+  # Only one driver per node is allowed, meaning we can we only dedicate 1 thread to this dispatcher
+  thread-pool-executor {
+    # Maximum number of threads within the pool.
+    # Supports running only 1 driver at a time
+    fixed-pool-size = 1
+  }
+}
+```
```

Comment:
Note that this configuration ref has new properties to consider for 2.1 i.e. it won't be a straight cherry pick.